bazci, roachtest: github template parameter

herkolategan opened the pull request:
Adding a template parameter to the GitHub issue creation URL is required to be
able to view the rendered template on GitHub, otherwise it just takes you to the
template selection page.

Epic: None
Release note: None

herkolategan committed:
roachtest: github template parameter

Adding a template parameter to the GitHub issue creation URL is required to be
able to view the rendered template on GitHub, otherwise it just takes you to the
template selection page.

Epic: None
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/143734)
<!-- Reviewable:end -->

herkolategan committed:
bazci: github template parameter

Adding a template parameter to the GitHub issue creation URL is required to be
able to view the rendered template on GitHub, otherwise it just takes you to the
template selection page.

Epic: None
Release note: None

DarrylWong reviewed (approved):
Thanks for fixing!

herkolategan commented:
TFTRs!

bors r=DarrylWong,rickystewart
